server/index.js

```diff
@@ -1,7 +1,7 @@
 const app = require('express')();
 const http = require('http').createServer(app);
 const io = require('socket.io')(http);
-const PORT = process.env.PORT || 3000;
+const PORT = process.env.PORT || 3001;
```

Comment:
We need to make a config file ignored by git to put our custom configurations without commit them